spaces/S000043/properties/P000064.md

```diff
@@ -0,0 +1,11 @@
+---
+space: S000043
+property: P000064
+value: true
+refs:
+- mathse: 476821
+  name: Sorgenfrey line is a Baire space
+---
+
+Shown in <https://www.jstor.org/stable/26453423>.
```

Comment:
I wanted to add this in the refs section, but there is no "doi" for it.  No big deal since there is at least the other (invisible) mathse ref, but any way to do this in general for a jstor ref?

Reply:
@StevenClontz Out of curiosity, do you know if there is a way to a have a jstor reference in the refs section?

Reply:
No, but I'll open an issue on referencing in general.